otj-metrics-core/src/main/java/com/opentable/metrics/SortedEntry.java

```diff
@@ -0,0 +1,76 @@
+package com.opentable.metrics;
+
+import java.util.Map;
+import java.util.TreeMap;
+
+import com.codahale.metrics.health.HealthCheck;
+import com.fasterxml.jackson.annotation.JsonValue;
+import com.google.common.collect.ComparisonChain;
+
+import org.apache.commons.lang3.ClassUtils;
+import org.apache.commons.lang3.builder.EqualsBuilder;
+
+import com.opentable.metrics.http.HealthController;
+
+public class SortedEntry implements Comparable<SortedEntry> {
+    final String name;
+    final HealthCheck.Result result;
+
+    public static Map<SortedEntry, HealthCheck.Result> render(boolean all, Map<String, HealthCheck.Result> raw) {
```

Comment:
Can you use a better parameter name than raw? Something that explains what the key and value of the map are.